tests/scripts/all.sh

```diff
@@ -5622,10 +5636,14 @@ component_build_armcc () {
     armc6_build_test "-O1 --target=aarch64-arm-none-eabi -march=armv8.2-a+crypto"
 }
 
-support_build_armcc () {
+support_build_armc5 () {
     armc5_cc="$ARMC5_BIN_DIR/armcc"
+    (check_tools "$armc5_cc" > /dev/null 2>&1)
```

Comment:
Preexisting: `check_tools` can use `return 1` instead of `exit 1` (I seem to have missed that years ago when structuring the files into functions), and so it wouldn't need to be put in a subshell here.